maintainers/maintainer-list.nix

```diff
@@ -3391,6 +3391,12 @@
     githubId = 2129135;
     name = "Frederik Rietdijk";
   };
+  friedelino = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add friedelino`` and sort it alphabetically.